codecs/webp_dec/.gitignore

```diff
@@ -0,0 +1,2 @@
+/*.wasm
```

Comment:
Weren't we going to start checking the wasm in?

Reply:
Yeah, that’s currently only in the testing branch. I’ll move that to master once this lands.